frame/staking/src/weights.rs

```diff
@@ -80,298 +80,325 @@ pub trait WeightInfo {
 /// Weights for pallet_staking using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	// Storage: Staking Ledger (r:1 w:1)
+	// Storage: Balances Locks (r:1 w:1)
+	// Storage: Staking HistoryDepth (r:1 w:0)
 	// Storage: Staking Bonded (r:1 w:1)
 	// Storage: Staking Payee (r:0 w:1)
-	// Storage: Staking HistoryDepth (r:1 w:0)
+	// Storage: Staking Ledger (r:1 w:1)
 	// Storage: Staking CurrentEra (r:1 w:0)
-	// Storage: Balances Locks (r:1 w:1)
 	fn bond() -> Weight {
-		(75_472_000 as Weight)
+		(76_854_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	// Storage: Staking Bonded (r:1 w:0)
-	// Storage: Staking Ledger (r:1 w:1)
+	// Storage: Staking VoterNodes (r:4 w:4)
+	// Storage: Staking VoterBagFor (r:1 w:1)
```

Comment:
thinking aloud - maybe we can eliminate this read if we store the current bag of the voter in the node

Reply:
yes let's merge them.

Reply:
sounds good - just to state the obvious, this will add a u64 (8 bytes) to each node. Storage space this should be less than the map since we don't need to store the key (accountId), but I think it means each node will take longer to decode